drivers/at/at.c

```diff
@@ -437,14 +449,23 @@ ssize_t at_send_cmd_get_lines(at_dev_t *dev, const char *command, char *resp_buf
     return -ENOBUFS;
 }
 
-int at_send_cmd_wait_prompt(at_dev_t *dev, const char *command, uint32_t timeout)
+ssize_t at_send_cmd_get_lines(at_dev_t *dev, const char *command, char *resp_buf,
+                              size_t len, bool keep_eol, uint32_t timeout)
 {
+    ssize_t res;
```

Comment:
seem like this could just be int;
at_send_cmd() returns int;
get_lines probaly returns an int to that it turned into an ssize;

this might be the other way around since some of the other functions also seem to return int even though they return size data

Reply:
atm this seems a little mixed

Reply:
should be `ssize_t` everywhere a size is returned but `isrpipe_read_timeout()` returns `int` too...

Reply:
right, so I changed everything like this:
 - fn that return 0 or negative error return `int`
 - fn that return a size or negative error return `ssize_t`
 - fn that return a size return `size_t`